src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/PermanentStorageConstants.java

```diff
@@ -0,0 +1,13 @@
+package org.fogbowcloud.saps.engine.core.archiver.storage;
+
+import java.io.File;
+
+public class PermanentStorageConstants {
```

Comment:
hmm ... these constants smell like an implementation detail. why do we need to expose them?

Reply:
Added fixme.